lib/Option.cpp

```diff
@@ -203,6 +193,28 @@ static llvm::cl::opt<bool> cluster_non_pointer_kernel_args(
                    "other arguments. Use this to reduce storage buffer "
                    "descriptors."));
 
+static llvm::cl::list<clspv::Option::StorageClass> no_16bit_storage(
+    "no-16bit-storage",
+    llvm::cl::desc("Specify fine-grained 16-bit storage capabilities. SSBO "
+                   "support is assumed."),
```

Comment:
"SSBO support is assumed" -> don't we assume all storage classes are supported by default? (possible hangover from when this was going to be an enabling option rather than a disabling one?)

Maybe "Specify" should also be changed to "Disable".